rllib/env/external_env.py

```diff
@@ -133,7 +133,7 @@ def log_action(self, episode_id, observation, action):
         episode.log_action(observation, action)
 
     @PublicAPI
-    def log_returns(self, episode_id, reward, info=None):
+    def log_returns(self, episode_id, reward, done=None, info=None):
```

Comment:
I think this could be confusing in single-agent mode, where you need to call end_episode() and setting done isn't sufficient. I guess this is true for multi-agent mode.

Open to ideas on how to avoid this, but what about not having `done` for single-agent ExternalEnv? For the policy client, since it is used for both single agent and multi-agent, we can have `multiagent_done_dict` as the param name.

Reply:
Yeah I had similar thoughts. I don't know what the exact approach is, how you guys see it. 
For me personally, I would like to always send done, for consistency. Better yet, when ending an episode done should be set to true and end_episode() should check this. However, ExternalEnv is transitioning quite heavily into something not "gym.env", or env for that matter at all. But yeah could go either way.

I would say let it depend on the future plans for ExternalEnv. If at some point the whole "gym.env" part is going to be redundant (like the fact that for serving you basically only define action and observation space), I would indeed remove the done. If looking for consistency with gym.env force the use of a done (dict) in all cases.

Reply:
How about we go with removing it in this case then? I don't think compatibility with gym.env is important, I'm more worried the two ways to set done will be unclear.